app/javascript/src/components/Dashboard/Notes/Collection/Card/index.jsx

```diff
@@ -0,0 +1,22 @@
+import React from "react";
+
+import Body from "./Body";
+import Footer from "./Footer";
+import Header from "./Header";
+
+const Card = ({ note }) => {
+  const { id, title, description, created_at } = note;
+
+  return (
+    <div
+      className="neeto-ui-shadow-s flex flex-col gap-4 rounded border border-gray-300 p-4"
+      key={id}
```

Comment:
Could you tell me why we need a key here?

Reply:
The key is not required here. Will remove it.